pkg/kubelet/dockertools/manager.go

```diff
@@ -1096,19 +1095,20 @@ func (dm *DockerManager) ExecInContainer(containerID kubecontainer.ContainerID,
 }
 
 func (dm *DockerManager) AttachContainer(containerID kubecontainer.ContainerID, stdin io.Reader, stdout, stderr io.WriteCloser, tty bool) error {
-	opts := docker.AttachToContainerOptions{
-		Container:    containerID.ID,
+	// TODO(random-liu): Do we really use the *Logs* field here?
```

Comment:
`Logs` is an argument in docker remote api https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#attach-to-a-container, go-dockerclient only needs to accept it and encoding it into json.